ClosedXML/Excel/CalcEngine/Functions/Text.cs

```diff
@@ -60,6 +62,22 @@ private static object Code(List<Expression> p)
             return (int)s[0];
         }
 
+        private static object Concat(List<Expression> p)
+        {
+            var sb = new StringBuilder();
+            foreach (var x in p)
+            {
+                if (x is IEnumerable enumerable)
+                {
+                    foreach (var i in enumerable)
+                        sb.Append((string)(new Expression(i)));
+                }
+                else
+                    sb.Append((string)x);
+            }
+            return sb.ToString();
```

Comment:
> If the resulting string exceeds 32767 characters (cell limit), CONCAT returns the #VALUE! error.

Don't we have to follow the spec?

Reply:
Erm, I'm in 2 minds about this... what does LibreOffice do?

Reply:
In LibreOffice Calc I just did a `CONCAT()` that return 64000 characters. So I would say this decision is implementation-specific.